### ISSUE TITLE
Probably not issue, just permissions heads-up

### ISSUE THREAD

**Issue by rpodric** [nextgens/anti-paywall#12]
In case it goes unnoticed, Chrome interpreted 0.8 as needing increased permissions (read and change all your data on the web sites you visit), so it automatically and silently disabled the extension.

The script is geared toward only certain web sites though, so I'm not sure why it said that. 

Installing it for the first time on Firefox Quantum requested similar permissions, though I don't know if it would do what's described above if I'd had it installed earlier.

**Comment by nextgens** [comment]
Yeah I've noticed; I made the change in c6cd7b842d1cd77d2764521f61e72452f1850e63
... precisely not to have to ask users to grant extra permissions at each update. 

I am not sure on how to deal with it: it's clearly sub-optimal to abuse the permission system that way... but the alternative (not being able to support new websites in new releases without having users re-enable the plugin and grant extra permissions) isn't good either...

Any thought on the topic would be appreciated. This is my first web-extension ^-^

**Comment by rpodric** [comment]
That seems a real Catch-22. This is probably an uncommon enough area that it wouldn't be surprising at all if Chromium has no accommodation for a middle ground.
